tests/chassis_modules_test.py

```diff
@@ -39,6 +40,70 @@
 SUPERVISOR0  192.168.1.100            True
 """
 
+show_chassis_system_ports_output_all="""\
+            System Port Name    Port Id    Switch Id    Core    Core Port    Speed
+----------------------------  ---------  -----------  ------  -----------  -------
+   Linecard1|Asic0|Ethernet0          1            0       0            1     100G
+   Linecard1|Asic0|Ethernet0          1            0       0            1     100G
```

Comment:
The same output lines are repeated, is this because the same system-port information is present in every asic ?
This output is be confusing.
Can we  either add
 -  a new column `asic_name` to indicate which asic this info is retrived or 
 - make the `-n <asicname>` mandatory for the multi asic linecards , so that the we don't display same system port information multiple times

Reply:
Yes. The same information will be repeated multiple times - as many times as the number of asics. This is because we have identical system port config in all asics. We display this multiple times so that one can verify for the requirement of indentical system port configuration in all asics.

The option to make the  <asic name> mandatory seems better than displaying all entries with new column since there will be lots of entries if there are many asics.

I'll implement changes to make the asic name mandatory in the command. Thanks for the suggestions.

Reply:
Fixed. Namespace option is made "required" option.